modules/javafx.controls/src/main/java/com/sun/javafx/scene/control/ControlAcceleratorSupport.java

```diff
@@ -242,7 +251,10 @@ public static void removeAcceleratorsFromScene(List<? extends MenuItem> items, N
             // The Node is not part of a Scene: Remove the Scene listener that was added
             // at the time of installing the accelerators.
             if (sceneChangeListenerMap.containsKey(anchor)) {
-                anchor.sceneProperty().removeListener(sceneChangeListenerMap.get(anchor));
+                ChangeListener<Scene> listener = sceneChangeListenerMap.get(anchor).get();
```

Comment:
Same as above: get() directly, don't check with containsKey().

Reply:
The conversation is below in the other reviewed code.